### PR TITLE
LRQA-82607

### PR DESCRIPTION
**Pull request by DavidOMontenegro**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            width1 = 1,
+            width2 = 080);
+
+		Click.waitForMenuToggleJSClick(locator1 = "ControlMenu#SIMULATION");
+		Click.waitForMenuToggleJSClick(locator1 = "ControlMenu#SIMULATION");
+
+		AssertElementPresent(locator1 = "ControlMenuPreviewPanel#PREVIEW_CUSTOM_SIZE_CONFIRMATION");
+		AssertTextEquals.assertPartialText(
+		locator1 = "ControlMenuPreviewPanel#PREVIEW_CUSTOM_SIZE_CONFIRMATION",
+		value1 = ${size});
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase:44]: Here is missing to pass the value1 after the changes from the last commit.

File: portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    @priority = "5"
+    test SetScreenSizeAndValidateValues {
+		MobileDevice.previewPGWithCorrectCustomNumbers(
+            height1 = 1,
+            height2 = 920,
+            width1 = 1,
+            width2 = 080);
+
+		Click.waitForMenuToggleJSClick(locator1 = "ControlMenu#SIMULATION");
+		Click.waitForMenuToggleJSClick(locator1 = "ControlMenu#SIMULATION");
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase:39]: Can you check if clicking twice is actually necessary? If not, we can adapt the steps or just remove this one.

File: portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase (hunk 1 above)
Comment (Luizcarlosqueiroz) [reply portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase:44]: Also, I investigated a bit deeper, and here you can use an existing path:
```
AssertTextEquals(
	locator1 = "Message#INFO",
	value1 = "Custom Size 1600x1600 Applied");
```

**Review by Luizcarlosqueiroz — CHANGES_REQUESTED** [review]
Hey @DavidOMontenegro ! I added some few comments here. Could you take a look?

File: portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase (hunk 2 above)
Comment (DavidOMontenegro) [reply portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase:39]: Clicking twice is necessary. The first click closes the menu and the second one reopens it.

Comment (Luizcarlosqueiroz) [reply portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase]: So maybe a solution is: create a if conditional to check if some element of the sidebar is not displayed. If true, then click the button. For example:
```
if (IsElementNotPresent(key_selectFieldLabel = "Product List Renderer", locator1 = "Select#ANY")) {
	Click(
		key_text = "Product List Renderer",
		locator1 = "Link#ANY");
}
```

Comment (DavidOMontenegro) [reply portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase]: The problem is the test is supposed to close and reopen it. That's what we're testing: If when I close the sidebar and reopen it, the values stay the same. A conditional like that would open the sidebar if it's closed (which it won't be) and not do anything if it's already open. That's not what I need. I need the sidebar to close and open again.

**Review by Luizcarlosqueiroz — APPROVED** [review]
LGTM!! :tada:

**Pull request closed without merging**